### PR TITLE
Jordan/notification copy updates

### PR DESCRIPTION
**Pull request by jbibla**
Closes #ISSUE

**Description:**
- added network to every notification
- improved grammar and wording

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: api/lib/notifications/notifications.js [hunk 1]
```diff
@@ -14,35 +14,25 @@ const config = require('../../config.js')
 
 function getMessageTitle(networks, notification) {
   const data = notification.properties || JSON.parse(notification.data)
+  const networkTitle = findNetworkTitle(networks, notification.networkId)
```
Comment (jbibla) [inline api/lib/notifications/notifications.js:17]: does this need to be a `let`?

Comment (faboweb) [reply api/lib/notifications/notifications.js:17]: no, why?

**Comment by faboweb** [comment]
Did you test this with an actual notification?

**Comment by faboweb** [comment]
Same copy will appear on the notifications page I guess

**Comment by faboweb** [comment]
🚀 Deployed on https://5f32f61235187dce1619630c--lunieio.netlify.app

**Comment by jbibla** [comment]
> no, why?

i wasn't sure exactly when the networkTitle would get set.

>  Did you test this with an actual notification?

no. i assumed it is a direct display of the content but i can take a look. is there a good address for viewing notifications (one that has a lot of the topics)?

File: api/lib/notifications/notifications.js [hunk 2]
```diff
@@ -73,37 +63,34 @@ function getMessageTitle(networks, notification) {
         2
       )}% (${percentageDifference.toFixed(2)}% decrease) for ${
         data.nextValidator.name
-      }`
+        } on ${networkTitle}`
     }
 
     case eventTypes.VALIDATOR_MAX_CHANGE_COMMISSION:
-      return `Max change for commisions has been updated from ${Number(
+      return `Max commission fee for ${data.nextValidator.name} has changed from ${Number(
```
Comment (Bitcoinera) [inline api/lib/notifications/notifications.js:70]: There are some linting warnings

**Comment by faboweb** [comment]
changelog missing

**Comment by jbibla** [comment]
> changelog missing

nope: https://github.com/luniehq/lunie/pull/4687/files#diff-0acfcaba02e963c5127423c3c504edcdR1

**Pull request merged**